### PR TITLE
Feature/room pictures db

### PR DESCRIPTION
**Pull request by jmettler27**
WIth this feature, the pictures of each room are now retrieved from the Firebase Storage.

### CODE REVIEW

File: app/src/androidTest/java/com/github/sdp/ratemyepfl/database/fakes/FakeImageStorage.kt [hunk 1]
```diff
@@ -1,26 +1,52 @@
 package com.github.sdp.ratemyepfl.database.fakes
 
+import com.github.sdp.ratemyepfl.R
 import com.github.sdp.ratemyepfl.database.Storage
 import com.github.sdp.ratemyepfl.model.ImageFile
+import com.github.sdp.ratemyepfl.utils.TestUtils.drawableToBitmap
 import javax.inject.Inject
 
 class FakeImageStorage @Inject constructor() : Storage<ImageFile> {
```
Comment (mbouilloud) [inline app/src/androidTest/java/com/github/sdp/ratemyepfl/database/fakes/FakeImageStorage.kt:9]: Do you still use this fake storage ?

Comment (mbouilloud) [reply app/src/androidTest/java/com/github/sdp/ratemyepfl/database/fakes/FakeImageStorage.kt:9]: Oh my bad, it's the test one :)

Comment (jmettler27) [reply app/src/androidTest/java/com/github/sdp/ratemyepfl/database/fakes/FakeImageStorage.kt:9]: Yes, in `RoomReviewPictureFragmentTest`

File: app/src/main/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivity.kt [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-import androidx.appcompat.app.AppCompatDelegate
 import com.github.sdp.ratemyepfl.R
 import kotlin.math.max
 import kotlin.math.min
 
 class ImageDetailActivity : AppCompatActivity() {
 
     companion object {
-        const val EXTRA_PHOTO_DISPLAYED: String = "com.github.sdp.extra_image_displayed"
+        var pictureDisplayed: Bitmap? = null
```
Comment (mbouilloud) [inline app/src/main/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivity.kt:16]: I think it's better to keep the old way i.e using an extra to pass the image id. Why did you change ?
I personally don't like the fact that you are using a var here.

Comment (mbouilloud) [reply app/src/main/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivity.kt:16]: Maybe the next reviewer could also give his opinion

Comment (jmettler27) [reply app/src/main/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivity.kt:16]: I don't like it either, in fact I tried in a first attempt to pass the bitmap in the Intent as recommended in https://stackoverflow.com/questions/11010386/passing-android-bitmap-data-within-activity-using-intent-in-android but I kept having errors. I finally followed an advice on this topic, with a static variable.

**Review by mbouilloud — APPROVED** [review]

**Review by mbouilloud — APPROVED** [review]
